HTML/mockup-mogo/js/renderHTML.js

```diff
@@ -0,0 +1,73 @@
+export const renderBlogs = (parent, typeOfBlog, serverData) => {
+  for (let i = 3; i < 6; i++) {
+    let child;
+
+    if (typeOfBlog === 'blog') {
+      child = `<div class="blog">
```

Comment:
better not to have html on js like here as it is hard to maintain and add something new, better option would use some kind of templates